src/scene/scene_lua.inl

```diff
@@ -166,35 +260,31 @@ std::vector<openspace::properties::Property*> findMatchesInAllProperties(
                     return;
                 }
 
-                // Check tag
-                if (isGroupMode) {
-                    const properties::PropertyOwner* matchingTaggedOwner =
-                        findPropertyOwnerWithMatchingGroupTag(prop, groupName);
-                    if (!matchingTaggedOwner) {
-                        return;
-                    }
+                // At this point we know that the property name matches, so another way
+                // this property to fail is if we provided a tag and the owner doesn't
+                // match it
+                if (isGroupMode && !ownerMatchesGroupTag(prop->owner(), groupName)) {
+                    return;
                 }
             }
             else if (!nodeName.empty()) {
-                size_t nodePos = uri.find(nodeName);
-                if (nodePos != std::string::npos) {
-                    // Check tag
-                    if (isGroupMode) {
-                        const properties::PropertyOwner* matchingTaggedOwner =
-                            findPropertyOwnerWithMatchingGroupTag(prop, groupName);
-                        if (!matchingTaggedOwner) {
-                            return;
-                        }
-                    }
-                    // Check that the nodeName fully matches the node in id
-                    else if (nodePos != 0) {
+                const size_t nodePos = uri.find(nodeName);
+                if (nodePos == std::string::npos) {
+                    return;
+                }
+
+                // Check tag
+                if (isGroupMode) {
+                    if (!ownerMatchesGroupTag(prop->owner(), groupName)) {
```

Comment:
Nitpicky, but this makes it so that this comment and the one below are on the same "reading level", if you know what I mean
```suggestion
                if (isGroupMode) {
                    // Check tag
                    if (!ownerMatchesGroupTag(prop->owner(), groupName)) {
```

Reply:
I'll leave it as it is since the `isGroupMode` check already belongs to the "Check tag" heading to me